hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStaleNodeHandler.java

```diff
@@ -50,6 +55,7 @@ public void onMessage(final DatanodeDetails datanodeDetails,
     super.onMessage(datanodeDetails, publisher);
     try {
       pipelineSyncTask.triggerPipelineSyncTask();
+      containerSizeCountTask.process(containerManager.getContainers());
```

Comment:
Have we verified if a node becomes stale, how container size will be impacted. Container may get closed on that DN, but some other replica for that container may be present in other DN in cluster. Pls check if we really need to call our task for StaleNode handler.

Reply:
The main function of the stale node handler is to ensure that the data stored in the pipelines associated with the stale node is not accessible anymore and also log the information about the stale node.

**Hence, since neither containers or their replicas are removed, the container size counts will not be affected. Therefore, we would not have to trigger the ContainerSizeCountTask.**